t/17-build_tagging.t

```diff
@@ -299,7 +299,7 @@ Then "important builds" are skipped from cleanup
 =cut
 subtest 'no cleanup of important builds' => sub {
     # build 0048 has already been tagged as important before
-    my $job = $jobs->search({id => 99938, state => 'done', group_id => 1001, BUILD => '0048'})->first;
+    my $job = $jobs->find({id => 99938, state => 'done', group_id => 1001, BUILD => '0048'});
```

Comment:
Maybe the test passes, but only coincidentally. See the other discussion. I will only do `WHERE (me.id = ? )`